WTWSMS/common/traits/VIET_Immersion_Other.txt

```diff
@@ -45,7 +46,7 @@ griot = { #West African Griot
 
 	customizer = no
 	
-	cannot_inherit = yes
+	cannot_inherit = no
```

Comment:
I'm not sure if that has always been the case, but all modern Griots can and do inherit. I found nothing indicating that they were like monks and were prohibited to inherit. Griots were basically African bards and historians, not religious figures.